monkey/common/user_roles.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class UserRoles(Enum):
+    """
+    An Enum representing user roles.
+    This Enum represents roles that the user can have. The value
+    of each member is the description of the role
```

Comment:
Have you considered making descriptions a separate dict? The benefit of having a separate dict is readability.

Reply:
Just a dictionary in the enum or anywhere the description is used?

Reply:
If this is an enum of descriptions it should be called that way. 
If this is an enum of roles we need to put descriptions elsewhere. So the value of a role should be `auto()` and we could have a dict:
```
RoleDescriptions = {
    IslandInterface: "Role for Island HTTP API user interfaces",
    Agent: "Role for the agents using the Islands' HTTP API"
}
```

Reply:
I think that is an overkill just for readability